virl2_client/models/groups.py

```diff
@@ -98,8 +98,8 @@ def create_group(
         """
         data = {
             "name": name,
-            "description": description,
-            "members": members or [],
+            **({"description": description} if description else {}),
+            **({"members": members} if members else {}),
             "labs": labs or [],
```

Comment:
Just add those keys in separate if statements after creating the base data IMO.

Reply:
members=members or None ... should work in that case, same thing for description if description can be None as well.

Reply:
Description cannot be none: 
```
GenericDescription:
  type: string
  # have no idea how our users use this field - do they use it at all
  maxLength: 4096
```
Also, on creation, setting labs or members to None does nothing since that is the default
So Tomas's suggestion is the only one that can be fully consistent

Reply:
Or @virlos's I guess, but that is an extra line per attibute! Think of the line economy!